### PR TITLE
LPS-131037 Send the portletId even when it is instanceable to filter deleted usages

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/layout/contents/contributor/LayoutClassedModelUsageLayoutContentsContributor.java [hunk 1]
```diff
@@ -232,6 +233,18 @@ private JSONObject _getActionsJSONObject(
 			).buildString());
 	}
 
+	private String _getIcon(String className) {
+		if (Objects.equals(FileEntry.class.getName(), className)) {
```
Comment (ealonso) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/layout/contents/contributor/LayoutClassedModelUsageLayoutContentsContributor.java:219]: @victorg1991 For this one, we should use the asset selector or even the info item api. Image the clients have another type, 🤔

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:stop

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/layout/contents/contributor/LayoutClassedModelUsageLayoutContentsContributor.java (hunk 1 above)
Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/layout/contents/contributor/LayoutClassedModelUsageLayoutContentsContributor.java:219]: Info framework doesn't support this at the moment, this is why I wanted to make it simple for now. Asset framework has something to retrieve the icon?

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**